### PR TITLE
Fixes Bandanas. Adds GAGS support for Bandanas

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request

Adds greyscale bandanas from tg. ~~Fixes the error sprite worn icon~~

Ports:
- https://github.com/tgstation/tgstation/pull/58400
- https://github.com/tgstation/tgstation/pull/65760

## Why It's Good For The Game

~~Fixes a broken icon~~, GAGS makes icon management less painful

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/user-attachments/assets/a08fe50d-1399-4f8b-a523-37ff75cb0b0a

</details>

## Changelog
:cl: rkz, Ebb-Real, AnturK
add: GAGSifies bandanas
refactor: refactors large mob icons to no longer need snowflaked .dmi sizes. Just edit a variable
/:cl:

### CODE REVIEW

File: code/_globalvars/bitfields.dm [hunk 1]
```diff
@@ -176,21 +176,22 @@ DEFINE_BITFIELD(resistance_flags, list(
 ))
 
 DEFINE_BITFIELD(flags_1, list(
```
Comment (Tsar-Salat) [inline code/_globalvars/bitfields.dm]: Just ordering alphabetically

**Comment by PestoVerde322** [comment]
What about the cage helmet? `icons/mob/large-worn-icons/64x64/head.dmi` is being removed after all and I see no mentions of it having a new icon adress. Same goes for other former 64x64 icons (which also are lacking on the :cl:). Other than that the icons for the bandana seems good to go.

**Review by PestoVerde322 — APPROVED** [review]
All good on the icon front now, approved!

**Comment by Tsar-Salat** [comment]
The newer fix has been merged, this is now pointless

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**